contracts/example/src/contract.rs

```diff
@@ -32,26 +32,11 @@ impl AxelarExecutableInterface for GmpExample {
 
 #[contractimpl]
 impl GmpExample {
```

Comment:
rename to Example. This will have both gmp and its example methods later

Reply:
updated.